### DISCUSSION TITLE
Any chance we get images-ext-2.discordapp.net/external custom popup to display images (forced download)?

### DISCUSSION THREAD

**Discussion by DevWannabe-dot** [qsniyg/maxurl#915] [Q&A]
https://images-ext-2.discordapp.net/external/D94oV46xlU7lQgkkySRcrxE2w3XrQeKx6k-S2dlxQgM/%3Fwidth%3D253%26height%3D333/https/media.discordapp.net/attachments/891727589402636298/892789346707583077/unknown.png for example, is useful for when an image is deleted but the thumb is still cached on the Discord image proxy

**Comment by qsniyg** [comment]
Sorry, I got to this too late and the link is now dead. I'm not sure I understand the question however. There is a rule for it, but are you wondering if IMU could use the inputted `images-ext-*.discordapp.net` url instead of the proxied URL? If so, it should automatically use that if the final URL doesn't load. Or does it not work?

**Comment by DevWannabe-dot** [comment]
Here is another link: https://images-ext-2.discordapp.net/external/IdDYzhQ6gkROFzWCOCqO_2U_UAW_sZdrv5HWwTacm20/https/lh3.googleusercontent.com/yzDtY44f0YDzDvX4rSZGVZfTIrAqNCEGkkh48_ZHBhE_6batho6j2TnWJX1fejtDHEW76feAnZ9K7QfbWomoUkhNP6mKnl4kckMh6vmwoh2huayBjpGO1VfsZXpfdBHJKz08WcRx%3Dw600-h315-p-k?width=400&height=210

I'm not sure if I was too overloaded at the time or something, because this `images-ext-2.discordapp.net/external` thingy actually refers to external images from other sources (googleusercontent, ytimg etc.), but you may notice the source forces the image download to display, so I was wondering if we could get the custom pop-up on this specific site? This might not be the smartest case but I'm convinced I had a more reasonable purpose in mind when I submitted the link that is now dead lol

**Reply by DevWannabe-dot** [reply]
because I had to use inspect on Discord app to find that example, which 99% people don't know Discord app is based on Chromium and they can inspect with Ctrl + Shift + I, but there are other places where links with that prefix may be found. Non-firefox users would sure be bummed at the forced download and maybe even ignore it

**Reply by qsniyg** [reply]
Sorry, I'm still not entirely sure I understand your question. Though it won't pop up on github due to CSP (unless partially loaded images are disabled, or the popup action is set to New Tab), it should be possible to open the popup with the URL proxied by images-ext-2.

**Comment by huutai97** [comment]
https://images-ext-2.discordapp.net
HOW CAN I USE THIS LINK ? with https://cdn.discordapp.com/attachments/867794882146992133/1149178053277270116/076e3nbfpsl81.png
Can some one help me ?

**Reply by qsniyg** [reply]
Sorry, I'm not sure what you're trying to do here. `images-ext-2.discordapp.net` is the domain name that Discord uses to proxy external images. The `cdn.discordapp.com` link you sent is already at the maximum resolution :)
